README.md

```diff
@@ -67,6 +67,30 @@ quest {uri: 'www.google.com', jar: j}, (err, resp, body) ->
   # The request to Google was sent with the cookie that you specified
 ```
 
+## Promises
+Quest also supports [ES6 Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise). However, make sure coffeescript is fully up to date. Older versions of coffeescript (e.g. 1.6.x) require parentheses for quest to correctly return Promise objects
+```coffeescript
```

Comment:
This doesn't seem like something that needs to be documented - I think the user of the library would be expected to write valid code in the language/version they're using. 

For example 
```
_ = require "underscore"
_ data
  .chain()
  .map(fn)
  .uniq()
  .value()
```

would have the same problem - it's not specific to promises or quest. 

Also quest is a js library - not always used in coffeescript. As such it seems even stranger to document a coffeescript issue in the readme. 

Thoughts?